frame/collective/src/lib.rs

```diff
@@ -211,7 +211,13 @@ decl_storage! {
 	add_extra_genesis {
 		config(phantom): sp_std::marker::PhantomData<I>;
 		config(members): Vec<T::AccountId>;
-		build(|config| Module::<T, I>::initialize_members(&config.members))
+		build(|config| {
+			let has_dupes = (1..config.members.len())
+				.any(|i| config.members[i..].contains(&config.members[i - 1]));
+			assert!(!has_dupes, "Members cannot contain duplicate accounts.");
```

Comment:
Would be more computationally efficient to use a hashset, but I am under the impression that might cause issues with non std environments; would appreciate if someone can confirm though :)

Reply:
You can use `BTreeSet`.

Reply:
Build spec must be std anyway. But BTreeSet is more commonly used in Substrate

Reply:
thanks. see ffea2a9 , which is based off of Kian's suggestioon.